gems/redis/4.2/_test/sorted_sets.rb

```diff
@@ -0,0 +1,6 @@
+redis = Redis.new
+
+redis.zcard(:sample_key)
+redis.zadd(:sample_key, 1, 'hello world')
```

Comment:
Can we test other cases? For example, `#zadd` can receive `:ch`, so it would be better to check the use of this option like this:

```ruby
redis.zadd("foo", 1, "s1", ch: true)
redis.zadd("foo", 1, "s1", incr: true)
redis.zadd("foo", 1, "s1", nx: true)
redis.zadd("foo", [2, "s1", 3, "s2", 4, "s3"], xx: true)
```

Reply:
I add some test case with optional keys.